--- Makefile	2019-09-30 17:23:13.349215000 -0700
+++ Makefile.cori	2019-09-30 17:22:57.130652000 -0700
@@ -1,7 +1,7 @@
 INSTALL = install
 RM = rm -f
 CC = gcc
-CFLAGS = -Wall -O2 -ggdb -std=gnu99
+CFLAGS = -Wall -O2 -ggdb -std=gnu99 -I/opt/cray/pe/papi/5.6.0.6/include -L/opt/cray/pe/papi/5.6.0.6/lib
 LD = $(CC)
 LDLIBS = -lm
 SYSTEM = $(shell uname -s | tr '[a-z]' '[A-Z]' | tr -d '_ -/')
@@ -39,7 +39,7 @@
 ifeq (x86_64,${ARCH})
     TARGET += libinterpose.so
     PAPI_SO=$(shell /sbin/ldconfig -p | grep libpapi.so)
-    PAPI_H=$(shell ls /usr/include/papi.h)
+    PAPI_H=$(shell ls /opt/cray/pe/papi/5.6.0.6/include/papi.h)
     ifneq ($(PAPI_SO),)
         ifneq ($(PAPI_H),)
             CFLAGS += -DHAS_PAPI
